tests/context/integration/test_repository.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import asdict
+
+from tests.poc.context_example.model import ExampleAggregate, ExampleEntity
+
+
+def test_aggregate_save(uow, raised_command):
+    entity = ExampleEntity.create(**asdict(raised_command))
```

Comment:
would be good to have the `ExampleVO` related to one of this to show that the `BaseRepository` can save Value objects as well

Reply:
Value Objects are transient objects, so they aren't stored.